src/UI/Navbar/NavbarLayoutTemplate.hbs

```diff
@@ -34,10 +34,12 @@
     </div><!--/.container-fluid -->
 
     <div class="col-md-12 search">
-        <div class="col-md-6 col-md-offset-3">
-            <div class="input-group">
-                <span class="input-group-addon"><i class="fa fa-search"></i></span>
-                <input type="text" class="col-md-6 form-control x-series-search" placeholder="Search the series in your library">
+        <div class="row">
```

Comment:
> Rows must be placed within a .container (fixed-width) or .container-fluid (full-width) for proper alignment and padding.

I believe that's why this fixes the width, because the padding is lost.

Reply:
So, this change is good, right?

Reply:
Probably better to be more explicit, instead of relying on the side effect of `row`.